example/image-classification/common/fit.py

```diff
@@ -238,7 +238,7 @@ def fit(args, network, data_loader, **kwargs):
             # AlexNet will not converge using Xavier
             initializer = mx.init.Normal()
             # VGG will not trend to converge using Xavier-Gaussian
-        elif 'vgg' in args.network:
+        elif args.network != None and 'vgg' in args.network:
```

Comment:
it would be cleaner and more pythonic if you did 'elif args.network and 'vgg' in args.network'

Reply:
Yes, that is vey correct. Thanks.